server/fcm_integration_test.go

```diff
@@ -55,6 +55,9 @@ type expectedValues struct {
 // Test that restarting the service continues to fetch messages from store for a subscription from lastID
 func TestFCMRestart(t *testing.T) {
 	//defer testutil.EnableDebugForMethod()()
+	//TODO BOGDAN COSMIN MARIAN modify the test to not use websocket client.
+	defer testutil.SkipIfDisabled(t)
```

Comment:
It is.Either we repair it and it will not give any other random fails and we re-enable it then.
Also this test uses websocket something which we do not use/need in gobbler.